src/js/routes/services.js

```diff
@@ -29,9 +32,34 @@ let serviceRoutes = {
           children: [
             {
               type: Route,
+              path: 'tasks/:taskID/?',
               name: 'services-task-details',
-              path: 'task-detail/:taskID/?',
-              handler: TaskDetail
+              handler: TaskDetail,
+              children: [
+                {
+                  type: Route,
+                  name: 'services-task-details-tab',
+                  path: 'details/?',
+                  handler: TaskDetailsTab
+                },
+                {
+                  type: Route,
+                  name: 'services-task-details-files',
+                  path: 'files/?',
+                  handler: TaskFilesTab
+                },
+                {
+                  type: Route,
+                  name: 'services-task-details-logs',
+                  path: 'logs/?',
+                  handler: TaskLogsTab
+                },
+                {
+                  type: Redirect,
+                  from: '',
```

Comment:
This isn't working correctly.
![](http://cl.ly/171s2x1F0G0X/Image%202016-06-08%20at%2011.36.00%20AM.png)